### PR TITLE
Fontsize soln

### PR DESCRIPTION
**Pull request by shrutimistry**
#### What does this implement/fix? Explain your changes.
Resolves the comments in pull request: https://github.com/VishwaP98/lets-git-it-scikit-learn/pull/2 
Adds a new file: bu3.py which can be used to execute this feature

### CODE REVIEW

**Review by linjun9 — COMMENTED** [review]
Please don't submit bu3 to the repo please, you can add it in the google doc or just have a Github gist.

File: sklearn/metrics/_plot/confusion_matrix.py [hunk 1]
```diff
@@ -508,6 +522,8 @@ def plot_confusion_matrix(estimator, X, y_true, *, labels=None,
 
     colorbar : bool, default=True
         Whether or not to add a colorbar to the plot.
+    
+    fontsize: int, default=
```
Comment (linjun9) [inline sklearn/metrics/_plot/confusion_matrix.py]: default = ?

Comment (shrutimistry) [reply sklearn/metrics/_plot/confusion_matrix.py]: removed it, it was unneccesary

File: sklearn/metrics/_plot/confusion_matrix.py [hunk 2]
```diff
@@ -29,6 +29,9 @@ class ConfusionMatrixDisplay:
     display_labels : ndarray of shape (n_classes,), default=None
         Display labels for plot. If None, display labels are set from 0 to
         `n_classes - 1`.
+    
+    font_size: int
```
Comment (linjun9) [inline sklearn/metrics/_plot/confusion_matrix.py]: you can add the default value here in the doc.

File: sklearn/metrics/_plot/confusion_matrix.py [hunk 3]
```diff
@@ -508,9 +522,7 @@ def plot_confusion_matrix(estimator, X, y_true, *, labels=None,
 
     colorbar : bool, default=True
         Whether or not to add a colorbar to the plot.
-
-        .. versionadded:: 0.24
-
+
```
Comment (linjun9) [inline sklearn/metrics/_plot/confusion_matrix.py]: why do you delete this line `.. versionadded:: 0.24`

**Review by linjun9 — APPROVED** [review]
it's good now. Do you want to squash some commits?

**Review by VishwaP98 — APPROVED** [review]

**Pull request merged**